<ui version="4.0" >
 <class>Imprimir</class>
 <widget class="QDialog" name="Imprimir" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Imprimir valor</string>
  </property>
  <widget class="QPushButton" name="aceptarButton" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelarButton" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>40</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelFrom" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>110</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>From:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Tolabel" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>170</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>to:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fromEdit" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>130</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="toEdit" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>190</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
